client/src/components/PacmanGame/PacmanGame.jsx

```diff
@@ -125,7 +125,24 @@ class PacmanGame extends Component {
   }
 
   startGame = () => {
-    const { config, pacman } = this.state;
+    const { config, pacman, status } = this.state;
+    if (status === 1) {
+      let { score } = this.state;
+      clearInterval(this.animationHandler);
+      score -= 10;
+      this.setState({ status: 0, score });
+      return;
+    }
+    if (status === 2) {
+      this.setInitialGameState();
+      this.setState({
+        score: 0,
+        moveGhostsCount: 0,
+      });
+      this.setState({ status: 1 });
+    }
+    if (status === 0) this.setState({ status: 1 });
```

Comment:
Replace the status code with 'string values'. It is hard to understand what status 1, 2, 3..etc mean.

Reply:
It's mentioned in the comment of the state in PacManGame Component.

Reply:
You wrote that comment btw :stuck_out_tongue:

Reply:
And the Strings will be on the Button